src/components/Table/Table.story.jsx

```diff
@@ -66,30 +66,36 @@ export const tableColumns = [
     id: 'string',
     name: 'String',
     filter: { placeholderText: 'pick a string' },
+    align: 'start',
```

Comment:
Please create a separate story to showcase this functionality and scope the feature to only that story instead of globally for all stories. Ideally this parameter would also be a knob so users can play with the different states of `start`, `end`, and `center`

Reply:
done.

Reply:
Did not incorporate a knob though